### PR TITLE
Commonjs stuff

### PR DESCRIPTION
**Pull request by isuda**
@dpogue

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+window.Ps = require('perfect-scrollbar');
```
Comment (dpogue) [inline index.js]: We're using CommonJS, but we're just not CommonJS enough to avoid exposing stuff on `window` :(

Comment (isuda) [inline index.js]: Well, I thought about this a bit. I could require it in the src file itself and avoid this window stuff. I am just not sure the best way to go about it.

Comment (isuda) [inline index.js]: OK, I will conditionally `require` instead.

**Comment by isuda** [comment]
updated.

**Comment by dpogue** [comment]
:+1:

**Pull request merged**